sys/vane/ames.hoon

```diff
@@ -1323,16 +1338,26 @@
       :_  fox  [hen %pass wire i.q.q.bon %west p.bon t.q.q.bon r.bon]~
     ::
         %ouzo
+      ::  drop packet if we haven't yet learned a unix duct
+      ::
+      ::     Only happens during first boot.
+      ::
+      ?~  gad.fox
+        [~ fox]
       ::  ~&  [%send now p.bon `@p`(mug (shaf %flap q.bon))]
-      ~|  [%ames-bad-duct duct=gad.fox lane=p.bon]
-      ?>  ?=(^ gad.fox)
       :_  fox
       [[gad.fox [%give %send p.bon q.bon]] ~]
     ::
         %pito
       :_  fox(tim `p.bon)
       %-  flop
       ^-  (list move)
+      ::  XX should this be the unix duct (gad.fox)?
```

Comment:
oof good question

I suspect we should make a specific constant duct whose sole purpose is this Ames-Behn interaction. 
`~[/ames-behn]` might be ok. I think we'd want it not to have a `//` at the front, to disambiguate it from ducts that have Unix origins.

If we do change that, though, I think we should do it in a different PR so we can git-bisect later if we need to.